### ISSUE TITLE
Klipper UI's printing estimates shows wrong total layer count

### ISSUE THREAD

**Issue by j-f-s** [QIDITECH/QIDI_Q1_Pro#14]
I notice lately that the total layer count in the klipper UI printing status mismatch what is shown in QIDISlicer's layer view.

![image](https://github.com/QIDITECH/QIDI_Q1_Pro/assets/91804108/1cc3be56-fb36-452d-ac4d-b1b223576d2b)
![image](https://github.com/QIDITECH/QIDI_Q1_Pro/assets/91804108/798694c4-6d33-4de1-999c-4195d43b97da)

The time estimate for "Finish" seems to be pretty close.

**Comment by CChen616** [comment]
Please refer to this issue here: https://github.com/fluidd-core/fluidd/issues/1403

The layer count displayed on the fluidd interface is an estimate, which I believe is based on the count of movements of the Z-axis. Currently, the version of Klipper used on the Q1 does not support the `SET_PRINT_STATS_INFO` macro. We are considering adding this feature in the next version.

**Comment by j-f-s** [comment]
I presume you're referring to [this commit from October 2022](https://github.com/Klipper3d/klipper/commit/b0ffb269d2ac8dc19ae6879d80718b8c60ab60de)?

**Comment by CChen616** [comment]
Yes exactly, you can modify `print_stats.py` file and restart klipper to enable the macro.
